desc/objectives/_coils.py

```diff
@@ -1110,6 +1111,11 @@ class QuadraticFlux(_Objective):
     vacuum : bool
         If true, B_plasma (the contribution to the normal field on the boundary from the
         plasma currents) is set to zero.
+    sqrt_area_weighting : bool
```

Comment:
does this need to be an option? I thought we would just always do this, so that the sum of squares is actually the integral of Bn^2

Reply:
I did not want to mess up any current code which has weights tuned considering the current funcitonaltiy. I am fine with changing this to be always True, but will have to re-do the coil opt tutorial a little

Reply:
I made the change here, but one of our CI actions fails bc the diff is too large if I include also the changes to the weigthts of the stage 2 tutorial, so I put those in #1342 which we can merge after this is in